frontend/src/libs/components/button/button.tsx

```diff
@@ -26,25 +28,32 @@ const Button: React.FC<Properties> = ({
   className,
   icon,
   isLabelVisuallyHidden = false,
-}: Properties) => (
-  <button
-    onClick={onClick}
-    type={type}
-    disabled={isDisabled}
-    className={getValidClassNames(
-      styles['button'],
-      styles[style],
-      styles[size],
-      className,
-    )}
-  >
-    {icon && <Icon iconName={icon} />}
-    <span
-      className={getValidClassNames(isLabelVisuallyHidden && 'visually-hidden')}
+  to,
+}: Properties) => {
+  const renderButton = (): JSX.Element => (
+    <button
+      onClick={onClick}
+      type={type}
+      disabled={isDisabled}
+      className={getValidClassNames(
+        styles['button'],
+        styles[style],
+        styles[size],
+        className,
+      )}
     >
-      {label}
-    </span>
-  </button>
-);
+      {icon && <Icon iconName={icon} />}
+      <span
+        className={getValidClassNames(
+          isLabelVisuallyHidden && 'visually-hidden',
+        )}
+      >
+        {label}
+      </span>
+    </button>
+  );
+
+  return to ? <Link to={to}>{renderButton()}</Link> : renderButton();
```

Comment:
```suggestion
  const content = (
        {icon && <Icon iconName={icon} />}
      <span
        className={getValidClassNames(
          isLabelVisuallyHidden && 'visually-hidden',
        )}
      >
        {label}
      </span>
  )

  return to ? <Link to={to}>{content}</Link> : <button>{content}</button>;
```
provide the same styles and other props

Reply:
Don't I need wrapp content inside link also with button to get it styled as button?

Reply:
You may just add styles to link to get it styled as button

Reply:
So it means to extend link component and let it have className prop, thx